README.md

```diff
@@ -70,6 +70,53 @@ Note that you should avoid importing styles from these modules, such as
 loaded by MediaWiki. Importing these styles would only cause additional
 overhead.
 
+### Development usage
+
+In Vite's `serve` mode, any import statements contianing modules in `using`
```

Comment:
Looks good! One last change requested: typo on `contianing`, and I'll merge this in. 😄